lib/fungus_toast/games/games.ex

```diff
@@ -50,9 +53,52 @@ defmodule FungusToast.Games do
 
   """
   def create_game(attrs \\ %{}) do
-    %Game{}
-    |> Game.changeset(attrs)
-    |> Repo.insert()
+    changeset = %Game{} |> Game.changeset(attrs)
+    user_name = Map.get(attrs, :user_name) || Map.get(attrs, "user_name")
+
+    with {:ok, game} <- create_game_for_user(changeset, user_name) do
+      human_player_count =
+        Map.get(attrs, :number_of_human_players) || Map.get(attrs, "number_of_human_players")
+      ai_player_count =
+        Map.get(attrs, :number_of_ai_players) || Map.get(attrs, "number_of_ai_players") || 0
+
+      game
+        |> Players.create_ai_players(ai_player_count)
+
+      game
+        |> set_new_game_status(human_player_count)
+    end
+  end
+
+  def create_game_for_user(_, "Fungusmotron") do
```

Comment:
Was this for testing something out? If so, does it still need to be in here?

Reply:
Fungusmotron is the user I created to generate AI players. It's not for a test. I'll probably switch it to ID 1